ark/utils/plot_utils.py

```diff
@@ -371,6 +371,28 @@ def create_overlay(fov, segmentation_dir, data_dir,
     return rescaled
 
 
+def set_minimum_color_for_colormap(cmap, default=(0, 0, 0, 1)):
```

Comment:
This is very nice @ackagel, I had to make a hacky version of this for the mesmer paper. This will be useful